lib/libutils/ext/qsort_helpers.c

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/*
+ * Copyright (c) 2024, STMicroelectronics
+ */
+
+#include <stdlib.h>
+#include <util.h>
+
+#define QSORT_HELPLER(name, type)				\
```

Comment:
I'm sorry, it seems that you've copied my typo. It should be `QSORT_HELPER`

Reply:
Indeed!!!